libraries/AP_HAL_ChibiOS/Storage.cpp

```diff
@@ -119,8 +165,27 @@ void Storage::_timer_tick(void)
     } 
 #endif
 
+#ifdef USE_POSIX
+    if (using_filesystem && log_fd != -1) {
+        uint32_t offset = CH_STORAGE_LINE_SIZE*i;
+        if (lseek(log_fd, offset, SEEK_SET) != offset) {
```

Comment:
Should we be checking for `-1`s being returned here?

Reply:
we're checking == offset, which is stronger than checking for not getting -1

Reply:
But we are not setting log_fd to -1 etc etc to indicate everything is stuffed - sorry, I wasn't clear.

Reply:
ahh, I want to allow for temporary failures. I don't want a single failure to cause a problem.